proto/ocpaths.proto

```diff
@@ -77,7 +77,8 @@ message OCPath {
 // fields for each type of constraint
 message OCPathConstraint {
   oneof constraint {
-    string platform_type = 1;
+    string platform_type = 1;  // derived from OC module oc-platform-types
+    string grpc_service = 2;   // derived from OC module oc-sys-grpc
```

Comment:
Do you want to list the identity that is actually being used to generate these values? (I assume that they are called `PLATFORM_TYPE` and `GRPC_SERVICE` but it might be clearer to explicitly state it?

Reply:
Good idea, I'll add

Reply:
Done